src/docs/guides/typed-components.md

```diff
@@ -5,20 +5,27 @@ url: /docs/typed-components
 contributors:
   - adamdbradley
   - manucorporat
+  - rwaskiewicz
 ---
 
 # Typed Components
 
-Web Components generated with stencil come with typing information automatically generated by the Stencil compiler. Typescript declarations provide strong guarantees when consuming components:
+Web Components generated with Stencil come with typing information automatically generated by the Stencil compiler.
```

Comment:
should we say `type declaration files` instead of `typing information`?

Reply:
👍 done in e1ce48cae